components/header/header.hbs

```diff
@@ -1,8 +1,25 @@
-<div class="container">
-  <header class="header">
-      <div class="logo header__logo">
-          <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg" alt="Office for National Statistics" class="logo__img">
-      </div>
-  </header>
-</div>
-{{>@bar bar_title="eQ Title bar"}}
+<header class="header {{classes}}">
+    <div class="header__top" role="banner">
```

Comment:
I think that it could be wise to remove `3px`? height from the `.header__top` and apply a 3px border to the bottom to provide a line for Zoomtext users. It would be there to guide them to the global navigation items.

Whether this should be a class that is appended when navigation is present so that it is not drawing attention to a blank space should be considered as well I think.